pages/flow.vue

```diff
@@ -28,7 +28,7 @@
             />
           </div>
           <div>
-            <a href="#consult">
+            <a href="#consult" v-scroll-to="'#consult'">
```

Comment:
v-scroll-toをhrefの前に書いていただきたいです🙏
```suggestion
            <a v-scroll-to="'#consult'" href="#consult">
```

Reply:
解消しました！